app/src/common/shared/com/igalia/wolvic/ui/widgets/settings/VoiceSearchLanguageOptionsView.java

```diff
@@ -44,7 +45,11 @@ protected void updateUI() {
         LayoutInflater inflater = LayoutInflater.from(getContext());
         SpeechRecognizer speechRecognizer = mWidgetManager.getServicesProvider().getSpeechRecognizer();
 
-        mSupportedLanguages = speechRecognizer.getSupportedLanguages();
+        if (speechRecognizer == null) {
+            mSupportedLanguages = Collections.emptyList();
+        } else {
+            mSupportedLanguages = speechRecognizer.getSupportedLanguages();
+        }
```

Comment:
as it's just an assignment I think we can just use the ternary operator and do it in one line

Reply:
Looks like you forgot this